site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+4.  To execute on this project, we will have to recruit partners and developers to not only help us understand specific requirements, but also to test experimental solutions at scale, provide feedback, iterate and improvise as and when required. The changes will have to be planned, allowing for a reasonable time frame for testing and evaluation.
+
+## Initial Standards Proposals
+
+We have performed some initial experiments to develop features that can be enabled to optimize the third-party loading process. We are pleased with the results observed and can presently discuss two of these features.
+
+1.  **LazyEmbeds**
+
+Chrome would previously [lazy-load](https://blog.chromium.org/2019/10/automatically-lazy-loading-offscreen.html) offscreen images and iframes for our [lite-mode](https://support.google.com/chrome/answer/2392284?hl=en&co=GENIE.Platform%3DAndroid) users. This feature could be extended to all users to defer the loading of iframes determined to be third-party embeds until the user scrolls near them. This could speed up the loading of other parts of the page, improve Core Web Vitals, reduce memory usage and save data.
+
+Here is a demo using LazyEmbeds to lazy load YouTube videos on a page. A single YouTube video embed typically adds 500-600KB of JavaScript to the page. We tried to optimize a [blog post](https://officeplaton.hatenablog.com/entry/2016/07/19/231529) with 14 such video embeds using LazyEmbeds. The results were promising for both data and time.
+
+<table>
+  <thead>
+    <tr>
+      <th></th>
+      <th><strong>Before</strong></th>
+      <th><strong>After</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Data</strong></td>
+      <td>15.4 MB</td>
+      <td>6.1 MB</td>
+    </tr>
+    <tr>
+      <td><strong>Total Blocking Time</strong></td>
+      <td>3.2 seconds</td>
+      <td>1.6 seconds</td>
+    </tr>
+  </tbody>
+</table>
+
+{% YouTube id='Ptf-eK3R3nA' %}
+
+To learn more about this work, see our Explainer and blink-dev [intent-to-experiment thread](https://groups.google.com/a/chromium.org/g/blink-dev/c/PhLkO3KITyw) and [experiment extension](https://groups.google.com/a/chromium.org/g/blink-dev/c/DjYa9xaoHhs).
+
+2.  **Targeted third-party Throttling**
+
+Third-party scripts are often added by various teams without holistic oversight processes. The engineering team at [The Telegraph](https://medium.com/the-telegraph-engineering/improving-third-party-web-performance-at-the-telegraph-a0a1000be5) articulated that "Everyone wants 'that tag' on a page which will make the organization money". This can continuously increase the burden of managing the performance impact.
```

Comment:
```suggestion
Third-party scripts are often added by various teams without holistic oversight processes. The engineering team at [The Telegraph](https://medium.com/the-telegraph-engineering/improving-third-party-web-performance-at-the-telegraph-a0a1000be5) articulated that "everyone wants 'that tag' on a page which will make the organization money". This can continuously increase the burden of managing the performance impact.
```